.babelrc

```diff
@@ -2,11 +2,15 @@
   "env": {
     "test": {
       "presets": [
-        ["es2015"],
-        ["stage-2"]
+        [
+          "env",
+          {
+            "targets": {"ie": 11}
+          }
+        ]
```

Comment:
Same for the others too:
```js
"presets": [
  ["env", {
    "targets": { "ie": 11 },
    "modules": false // where needed
  }]
],
```

Reply:
I'm not sure if I've got it. ES modules is not recommended in Jest and using it for the ES5 version makes it no longer ES5.

Reply:
Just the formatting, keep the modules as you have it.